.github/workflows/doc_build.yml

```diff
@@ -0,0 +1,35 @@
+name: Docs build
```

Comment:
we check more than just docs, we also test the most recent `easybuild` installation end-to-end